packages/amplify_authenticator/lib/src/widgets/form_fields/confirm_sign_in_form_field.dart

```diff
@@ -217,6 +251,55 @@ abstract class _ConfirmSignInFormFieldState<FieldValue>
   }
 }
 
+class _ConfirmSignInPhoneField extends ConfirmSignInFormField<String> {
```

Comment:
copy/paste from SignUp. Will look into reducing duplicated code in a follow up PR.